lua/mason-nvim-dap/mappings/source.lua

```diff
@@ -12,7 +12,6 @@ M.nvim_dap_to_package = {
 	['firefox'] = 'firefox-debug-adapter',
 	['php'] = 'php-debug-adapter',
 	['coreclr'] = 'netcoredbg',
-	['js'] = 'js-debug-adapter',
```

Comment:
could this be kept then this change wouldn't be breaking? it would be useful to add a vim.deprecate but i can do that later

Reply:
sure thing, on it

Reply:
done 91e37c7d190c831ce20490b83baee47ee65eef74

Reply:
sorry, i am dumb. this can't be kept because of the inversion that happens below. it needs to be deleted argh sorry for this

Reply:
good point, I'll remove it then

Reply:
fixed 26390ce5840db088601e437ccfa7c015d62e3a14

Reply:
I think this inversion is a wrong approach.

Particularly because some packages add multiple adapters, like `js-debug-adapter` can add multiple adapters.
Right now, we are only adding one adapter (which is actually deprecated in VSCode - as it prefers the name "node" above "pwa-node" ) but there are more which we may need.